server/eclipse-project/src/main/java/us/freeandfair/corla/json/CountyDashboardRefreshResponse.java

```diff
@@ -202,10 +208,20 @@
    * @param the_optimistic_ballots_to_audit The optimistic ballots to audit.
    * @param the_ballots_remaining_in_round The ballots remaining in the 
    * current round.
-   * @param the_cast_ballot_count The number of ballots cast.
+   * @param the_ballot_manifest_count The number of ballots represented by the
+   * uploaded ballot manifest.
+   * @param the_cvr_export_count The number of CVRs in the uploaded export file.
    * @param the_audited_ballot_count The number of ballots audited.
+<<<<<<< HEAD
```

Comment:
Merge conflict artifacts here and below.

Reply:
Got 'em.  7fba64b2